db/rdbms/src/main/java/io/camunda/db/rdbms/write/domain/VariableDbModel.java

```diff
@@ -7,11 +7,94 @@
  */
 package io.camunda.db.rdbms.write.domain;
 
+import org.apache.commons.lang3.math.NumberUtils;
+
 public record VariableDbModel(
     Long key,
-    Long processInstanceKey,
-    Long scopeKey,
     String name,
+    double varDouble,
+    long varLong,
```

Comment:
🔧 I feel like `<type>Value` naming is better.
❓ Shouldn't these fields be defined boxed primitives as one of them will always be null?

```suggestion
    Double doubleValue,
    Long longValue,
```

Reply:
Yes, good idea, I will change it